e2e/LongHaul/device/Program.cs

```diff
@@ -128,6 +133,18 @@ private static Logger InitializeLogging(Parameters parameters)
             return logger;
         }
 
+        private static int PortFilter(Parameters parameters)
+        {
+            return parameters.TransportProtocol == IotHubClientTransportProtocol.WebSocket
```

Comment:
The problem with this is that even if we choose to use AMQP/MQTT and TCP/WebSocket here, HTTP operations on the client will still be on a different port.